packages/calling/calling/src/CallingClient/CallingClient.ts

```diff
@@ -585,6 +234,44 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
       }
     );
   }
+
+  /**
+   *
+   */
+  private createLine() {
+    const lineId = uuid.v4();
+
+    const lineInstance = new Line(
+      // dummy values
+      this.webex.internal.device.userId,
+      this.webex.internal.device.url,
+      lineId,
+      '131212345',
+      '212343435',
+      LineStatus.INACTIVE,
+      ['add1', 'add2'],
+      '321232343',
+      '00',
+      100,
+      200,
+      50,
+      20,
+      100100100,
+      10,
+      this.callManager,
+      this.mutex,
+      this.sdkConfig
+    );
+
+    this.lineDict[lineId] = lineInstance;
+  }
+
+  /**
```

Comment:
Please add a description here.

Reply:
as said above, we won't be adding jsdocs from now onwards. We will just have a brief description of what the function is doing. Will add it.

Reply:
Added description.